packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-VisualBasic/App.xaml.vb

```diff
@@ -0,0 +1,6 @@
+﻿Class App
```

Comment:
Thanks for adding this, @Berrysoft.  

Could we make this more congruent to the .NET Framework Visual Basic project?   Application.xaml, 'class Application', etc.  Thanks.

Reply:
@Berrysoft, I just realized that you are a first-time contributor - welcome and thanks for contributing! 

We just had an internal meeting about adding VB templates yesterday and had inadvertently assumed that you were a contributer from within Microsoft. You probably don't have any context about some of my questions. We will help you through some of these steps...

Reply:
@ryalanms Unfortunately, this class must be named App otherwise we'll run into the issue described in #500.

Reply:
I've just filed #545 to solve the issue I mentioned above. Once that PR has been merged, we can use `Application.xaml` in the VB template.

Reply:
Thanks, @wjk.  We will take your change first.

Reply:
I changed `App` to `Application` just now.